src/components/TextInput/index.spec.js

```diff
@@ -0,0 +1,81 @@
+import { describe, expect, it } from 'vitest';
+import { render, screen, userEvent } from '@testing-library/vanilla';
```

Comment:
```suggestion
import { render, screen, userEvent } from '@testing-library/react';
```

Aqui tem que ser vanilla mesmo?

Reply:
import { render, screen, userEvent, waitFor } from '@testing-library/react';

Reply:
Tem sim nico, no pet a gente não usa react, a gente usa vanillaJs